utils/generate_enumerations.py

```diff
@@ -27,6 +27,14 @@ def format_enumeration_values(values: List[EnumerationEntry], kind: EnumKind) ->
     return f"\n{indentation}".join(result)
 
 
+def format_enumeration_literal_values(values: List[EnumerationEntry], kind: EnumKind) -> str:
```

Comment:
I will remove , `kind: EnumKind` because it will always be a string enum.
I tought to enable it for numbers but there is no value in `Literal[1, 2, 3, 4]` types.